conda/dali_bindings/recipe/build.sh

```diff
@@ -0,0 +1,119 @@
+#!/bin/bash
+#
+# (C) Copyright IBM Corp. 2019. All Rights Reserved.
+# (C) Copyright NVIDIA CORPORATION. 2019. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+#Determine Architecture
+
+ARCH="$(arch)"
+if [ ${ARCH} = "x86_64" ]; then
+    ARCH_LONGNAME="x86_64-conda_cos6"
+elif [ ${ARCH} = "ppc64le" ]; then
+    ARCH_LONGNAME="powerpc64le-conda_cos7"
+else
+    echo "Error: Unsupported Architecture. Expected: [x86_64|ppc64le] Actual: ${ARCH}"
+    exit 1
+fi
+
+# Create 'gcc' and 'g++' symlinks so nvcc can find it
+ln -s $CC $BUILD_PREFIX/bin/gcc
+ln -s $CXX $BUILD_PREFIX/bin/g++
+
+# Force -std=c++17 in CXXFLAGS
+export CXXFLAGS=${CXXFLAGS/-std=c++??/-std=c++17}
+
+# For some reason `aligned_alloc` is present when we use compiler version 5.4.x
+# Adding NO_ALIGNED_ALLOC definition for cutt
+export CXXFLAGS="${CXXFLAGS} -DNO_ALIGNED_ALLOC"
+export PATH=/usr/local/cuda/bin:${PATH}
+
+# For some reason `aligned_alloc` is present when we use compiler version 5.4.x
+# Adding NO_ALIGNED_ALLOC definition for cutt
+export CXXFLAGS="${CXXFLAGS} -DNO_ALIGNED_ALLOC"
+export PATH=/usr/local/cuda/bin:${PATH}
+
+# make it on by default for CUDA 11.x
+if [ "${CUDA_VERSION/./}" -ge 110 ]; then
+  export WITH_DYNAMIC_CUDA_TOOLKIT_DEFAULT=ON
+else
+  export WITH_DYNAMIC_CUDA_TOOLKIT_DEFAULT=OFF
+fi
+
+
+export BUILD_NVCOMP=${BUILD_NVCOMP:-OFF}
+
+# Create build directory for cmake and enter it
+mkdir $SRC_DIR/build_bindings
+cd $SRC_DIR/build_bindings
+
+# allow DALI import all dependencies in the build env
+export LD_LIBRARY_PATH="$PREFIX/libjpeg-turbo/lib:$PREFIX/lib:$LD_LIBRARY_PATH"
+
+# Build
+cmake -DCUDA_TOOLKIT_ROOT_DIR=/usr/local/cuda \
```

Comment:
is /usr/local/cuda referring to some conda environment?

Reply:
We still use the CUDA installed outside of the conda. The one we use for the regular build. While we can easily install CUDA in conda, installing nvJPEG2000 and nvCOMP is not that straightforward. So using the same CUDA env from the regular build allows us to maintain this composition logic only in one place for now.